sources/packages/web/src/types/AppRoutes.ts

```diff
@@ -39,6 +39,7 @@ export enum AppRoutes {
   DesignationRequest = "manage-designation/request",
   DesignationView = "manage-designation/view/:designationAgreementId",
   InstitutionManageUsers = "manage-users",
+  SubmittedApplicationScholasticStandingView = "submitted-application/scholastic-standing/:scholasticStandingId",
```

Comment:
Are use using this `submitted-application` anywhere else? It does not seem accurate.